### DISCUSSION TITLE
Do you have to name content files "page.md" otherwise they are not found?

### DISCUSSION THREAD

**Discussion by JetGum** [datenstrom/community#579] [Ask a question] [answered]
Dears,

sorry for the complicated title...  

With update to Yellow Core 0.8.16, I found an obscure error when content files are not named "page.md"  - Yellow does not display the content ("Sorry, file not found" error).

Example: I have a file in content/1-home which is called "home.md". If renamed to "page.md", Yellow works fine.
I have several installation of Yellow online and all rely on "speaking" filenames, which makes editing content a lot easier. 

Cheers,

Carsten

**Comment by markseuffert** [comment]
Yes, you're right.

You have to name your default files `page.md`. 

I am sorry to say, alternative names for default files have been removed. There were two reasons for this change. First for practical reasons, when you rename a folder in the file system, the default file `page.md` is always working. Second for technical reasons, when you rename a folder in the web browser, the underlying code can be much simpler, as it doesn't need to check for possible file name collisions.  I stumbled across this when adding support to restore deleted pages in the web browser. I didn't want that by deleting or un-deleting a file, another one is elevated to default file. Probably doesn't happen very often, but the devil is in the details.  

Carsten, is it possible to rename your default files? Do you have a lot of them?

**Comment by JetGum** [comment]
Hi, 

I have the content on my sites organized in folders and subfolders to create a multi-level navigation. Just to be sure I understand this correctly: I need to change all filenames to page.md? No more "speaking" filenames?

Cheers,

Carsten

**Comment by markseuffert** [comment, marked as answer]
No, you don't need to rename all content files. Just make sure that every folder has a file called `page.md`. You can add more files and folders. Here are some examples:

The file `content/9-about/page.md` becomes `http://website/about/`.
The file `content/9-about/about.md` becomes `http://website/about/about`.
The file `content/9-about/privacy.md` becomes `http://website/about/privacy`.

In any case, you can no longer use alternative names or "speaking names" for `page.md`.
